hooks/ingressroute_configurator/Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM golang:1.15-alpine
+# enable Go modules support
+ENV GO111MODULE=on
+RUN mkdir /src
+WORKDIR /src
+COPY ./go.mod .
+COPY ./go.sum .
+RUN go mod download
+ADD . /src/
+RUN CGO_ENABLED=0 GOOS=linux go build -a -o bin/irc main.go
```

Comment:
Do we need to add a user for running the entrypoint? Looks a bit insecure to run as root